librocksdb-sys/build.rs

```diff
@@ -111,41 +129,46 @@ fn build_rocksdb() {
     }
 
     if target.contains("darwin") {
-        config.define("OS_MACOSX", Some("1"));
-        config.define("ROCKSDB_PLATFORM_POSIX", Some("1"));
-        config.define("ROCKSDB_LIB_IO_POSIX", Some("1"));
+        config.define("OS_MACOSX", None);
```

Comment:
None means `-DOS_MACOSX` ?

Reply:
yes，pass None as the value for a value-less define.
https://rust-lang-nursery.github.io/rust-cookbook/development_tools/build_tools.html#compile-a-c-library-while-setting-custom-defines.

Reply:
this fetch from upstream.